app/src/main/java/com/asfoundation/wallet/transactions/PerkBonusAndGamificationService.kt

```diff
@@ -156,20 +156,28 @@ class PerkBonusAndGamificationService :
       val totalAppCoinsAmountThisLevel =
         statsPromotions.nextLevelAmount!!.minus(currentLevelStartAmount)
       val currentAppCoinsAmountThisLevel = statsPromotions.totalSpend.minus(currentLevelStartAmount)
-      if (  // TODO differentiate notifications for vips
-        statsPromotions.gamificationStatus == GamificationStatus.APPROACHING_NEXT_LEVEL ||
-        statsPromotions.gamificationStatus == GamificationStatus.APPROACHING_VIP ||
-        statsPromotions.gamificationStatus == GamificationStatus.APPROACHING_VIP_MAX
-      ) {
-        promotionsRepository.shownLevel(address, currentLevel, NOTIFICATIONS_ALMOST_NEXT_LEVEL)
-        buildNotification(
-          createAlmostNextLevelNotification(
-            formatter.formatGamificationValues(
-              totalAppCoinsAmountThisLevel
-                .minus(currentAppCoinsAmountThisLevel)
-            )
-          ), NOTIFICATION_SERVICE_ID_ALMOST_LEVEL_UP
-        )
+      promotionsRepository.shownLevel(address, currentLevel, NOTIFICATIONS_ALMOST_NEXT_LEVEL)
```

Comment:
Previously this method was called only for `Approaching ...` gamification statuses.
Now it is called always.
Is it a desired behaviour?